Directory.Packages.props

```diff
@@ -38,5 +38,6 @@
     <GlobalPackageReference Include="PolySharp" Version="1.14.1" />
     <GlobalPackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
     <GlobalPackageReference Include="Speckle.InterfaceGenerator" Version="0.9.6" />
+    <GlobalPackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3" PrivateAssets="All" />
```

Comment:
Don't need `PrivateAssets` since you're using `GlobalPackageReference` https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management#global-package-references